src/components/variables.ts

```diff
@@ -1,26 +1,25 @@
-//colors
 export type Color = keyof typeof palette;
 
 export const palette = {
-  purple: '#4A2ACB',
-  midpurple: '#5C33FF',
-  darkpurple: '#37119D',
-  palepurple: '#E5E4FF',
-  hoverpurple: '#CDCBFF',
-  bluegrey: '#4C6282',
-  blue2: '#43BFFF',
-  darkblue: '#1C325D',
-  lightblue: '#D3EFFF',
-  whiteblue: '#E4F3FB',
-  footerblue: '#01A5EC',
+  backgroundBlur: 'rgba(57, 57, 57, 0.75)',
+  backgroundSpinner: '#DDEBE9',
+  blue: '#43BFFF',
+  blueDark: '#1C325D',
+  blueFooter: '#01A5EC',
+  blueLight: '#D3EFFF',
+  greyBluish: '#4C6282',
+  greyFaded: '#616161',
+  greyLight: '#E9E9E9',
+  greyMid: '#DBDBDB',
   orange: '#F0BA8C',
   orange2: '#FFD79B',
+  purple: '#4A2ACB',
+  purpleDark: '#37119D',
+  purpleHover: '#CDCBFF',
+  purpleMid: '#5C33FF',
+  purplePale: '#E5E4FF',
```

Comment:
I like the new naming.
The old names were from Figma, so I would also want to rename the Figma-palette too. So its easier to refer later.

Reply:
That would be nice 🤔 Allthough I didn't find specific names for all of them.

Reply:
Hmm, the palette is a bit difficult to find in the UI of Figma.  Maybe we could just connect the color to the name. So just make sure that the names are same in both =P

Reply:
Ah yes, I found it. I'll go through it and get back to this asap!